org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -2803,6 +2809,57 @@ public void afterClose(Trx trx) {
 				else if (get_ID() > 0)
 					Adempiere.getThreadPoolExecutor().submit(() -> CacheMgt.get().newRecord(p_info.getTableName(), get_ID()));
 			}
+		} else if (p_info.getTableName().endsWith("_Trl") && CacheMgt.get().hasCache(TRANSLATION_CACHE_TABLE_NAME) && !newRecord) {
+			MTable table = MTable.get(getCtx(), p_info.getTableName().substring(0, p_info.getTableName().length() - 4));
+			POInfo parentInfo = POInfo.getPOInfo(getCtx(), table.getAD_Table_ID());
+			// take all translated columns, is_valuechanged(columnName) not working at this point
```

Comment:
a list of changed columns could be obtained around line 2763, it's too aggressive to reset all columns as it will take extra time making more dangerous to get an invalid cache